conans/model/conf.py

```diff
@@ -117,6 +117,8 @@
     "tools.apple:enable_arc": "(boolean) Enable/Disable ARC Apple Clang flags",
     "tools.apple:enable_visibility": "(boolean) Enable/Disable Visibility Apple Clang flags",
     "tools.env.virtualenv:powershell": "If it is set to True it will generate powershell launchers if os=Windows",
+    "tools.env.virtualrunenv:autogenerate": "Whether to autogenerate virtualrunenv launchers for consumer recipes (True by default)",
+    "tools.env.virtualbuildenv:autogenerate": "Whether to autogenerate virtualbuildenv launchers for consumer recipes (True by default)",
```

Comment:
Maybe try in command line argument ``--no-env`` instead of conf-driven?
This affect only ``conan install`` not other commands.

Reply:
We need to think about possible different behaviors wanted, like generating only the env-scripts if they are not empty. So maybe a cli arg like ``--generate-env=True|False|NotEmpty|...`` would be good.


It would be great to push the feature of not generating the deactivate scripts in the user folder, that will also reduce friction.